components/survey/EmptyCard/index.jsx

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+
+import Image from '@/components/Image';
+import sunglasses from '@/public/sunglasses.svg';
+
+export const emptySurveyCardTestIds = {
+  base: 'empty-survey-card',
+  sunglassesIcon: 'empty-survey-card__sunglasses-icon',
+  surveyCompletedText: 'empty-survey-card__survey-completed-text',
+};
+
+const EmptySurveyCard = ({ setBackgroundImagePath }) => {
+  useEffect(() => {
+    setBackgroundImagePath('/dark.png');
```

Comment:
[next/image](https://nextjs.org/docs/api-reference/next/image) provides image optimization and requires a `src` prop to render an image. As. we are using this `next/image` everywhere in this project, this implementation keeps it consistent.

Reply:
I see, I just think that maybe it can be a small improvement on the `Background` component when there is no image path or a blank string passing to it then it can render a div with gradient background instead 💡

Reply:
I think it's a nice idea to pass a blank string to `src` and render a dark background, done in 0a8596544c05464d0eef88743810267b02504566, Thank you!